conda.recipe/meta.yaml

```diff
@@ -30,6 +30,13 @@ test:
     {% for dep in sdata['extras_require']['tests'] %}
     - "{{ dep }}"
     {% endfor %}
+    - nose
+  commands:
+    - nosetests -vv --nologcapture
```

Comment:
I don't think we are using nose anywhere any more; please update to be like panel.